### PR TITLE
Syndicate Bomb (THE DRINK OBVIOUSLY) V2.0 the right way

### PR DESCRIPTION
**Pull request by Therandomhoboo**
# Document the changes in your pull request

This is for the Syndicate Bomb drink, but done right cause I'm learning how to code. All antag drinks do something for their respective antags about for the syndicate bomb, here it is. A small heal buff

Again, this is my first proper PR. Any tips or feedback would be appreciated. 

# Wiki Documentation

Two glasses connected with a tank valve. You need to drink from the valve, else it spills. If you're part of the glorious Syndicate, this heals brute & burn damage for 0.5 in all body parts per tick.

# Changelog

:cl:  
tweak: Syndicate bomb heals brute & burn for 0.5 per tick in all bodyparts for people that are part of the Syndicate
/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
erm akthally, this is your second pr now

**Comment by Therandomhoboo** [comment]
> erm akthally, this is your second pr now

BITE ME ITS BASED ON MY FIRST PR BUT I FUCKED IT UP

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2418,7 +2420,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "glass of kortara"
 	glass_desc = "The fermented nectar of the Korta nut, as enjoyed by lizards galaxywide."
 
-	
+
```
Comment (MajManatee) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: what's with the added spaces?

Comment (Therandomhoboo) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: is it where I put in the extra code? Ill go through and delete it

Comment (Therandomhoboo) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I've deleted the extra space

File: yogstation.dme [hunk 2]
```diff
@@ -1467,6 +1467,7 @@
 #include "code\modules\admin\view_variables\topic_basic.dm"
 #include "code\modules\admin\view_variables\topic_list.dm"
 #include "code\modules\admin\view_variables\view_variables.dm"
+#include "code\modules\antagonists\sentient_creature.dm"
```
Comment (Moltijoe) [inline yogstation.dme]: ?

Comment (Therandomhoboo) [reply yogstation.dme]: I ddont know why that has been added. The only code i added was for the drinks? Should I put it back in the line it was deleted from?

Comment (Moltijoe) [reply yogstation.dme]: it's usually best if only lines pertaining to the PR are touched

Comment (Therandomhoboo) [reply yogstation.dme]: All good, I don't know why this happened since the Syndicate Bomb was the only code I did touch. But it has been fixed now.

**Comment by TheBombMakerJL** [comment]
syndicate antag test

**Comment by Therandomhoboo** [comment]
> syndicate antag test

There are five other drinks that can be "antag-test" such as 
Wizz fizz- Heals Wizard
Nar'sour- Heals blood cults
Justicar- Heals Clock cults
Changeling sting- Regens Chemicals
and finally Cuba Libre- Heals revolutionaries

Drinks aren't a form of accurate test and it can't prove that they're Antag since they breaks metashield.

**Comment by TheBombMakerJL** [comment]
> > syndicate antag test
> 
> There are five other drinks that can be "antag-test" such as Wizz fizz- Heals Wizard Nar'sour- Heals blood cults Justicar- Heals Clock cults Changeling sting- Regens Chemicals and finally Cuba Libre- Heals revolutionaries
> 
> Drinks aren't a form of accurate test and it can't prove that they're Antag since they breaks metashield.

Wizard -> Loud Antag/Doesn’t benefit from a test like drinking
Blood Cult -> Loud Antag/Doesn’t benefit from a test like drinking 
Clock Cult -> Loud Antag/Doesn’t benefit from a test like drinking
Revs -> Loud Antag/Doesn’t benefit from a test like drinking 
Traitor -> Stealth Antag until traitor goes loud/Security will benefit from a test like drinking

+ 
Information like this would be posted on guide to bartending which is available ICly, and no information on the effects of these drinks are metashielded

**Comment by Maia-J** [comment]
> > There are five other drinks that can be "antag-test" such as Wizz fizz- Heals Wizard Nar'sour- Heals blood cults Justicar- Heals Clock cults Changeling sting- Regens Chemicals and finally Cuba Libre- Heals revolutionaries
> > Drinks aren't a form of accurate test and it can't prove that they're Antag since they breaks metashield.
> 
> Wizard -> Loud Antag/Doesn’t benefit from a test like drinking Blood Cult -> Loud Antag/Doesn’t benefit from a test like drinking Clock Cult -> Loud Antag/Doesn’t benefit from a test like drinking Revs -> Loud Antag/Doesn’t benefit from a test like drinking Traitor -> Stealth Antag until traitor goes loud/Security will benefit from a test like drinking
> 
> 
> Information like this would be posted on guide to bartending which is available ICly, and no information on the effects of these drinks are metashielded

Since when are Culties and Revs inherently loud? Their beginnings are stealth and they usually fall flat if they go loud too soon. Also you kinda missed changelings.

**Comment by TheBombMakerJL** [comment]
> Since when are Culties and Revs inherently loud? Their beginnings are stealth and they usually fall flat if they go loud too soon. Also you kinda missed changelings.

I didn't miss it, he didn't mention it because changelings don't heal from changeling sting. 
Security doesn't benefit from a cult/rev drink test because they already have a means of deconversion which they will always use over some drink and cultists have their gear

**Comment by Therandomhoboo** [comment]
> > Since when are Culties and Revs inherently loud? Their beginnings are stealth and they usually fall flat if they go loud too soon. Also you kinda missed changelings.
> 
> I didn't miss it, he didn't mention it because changelings don't heal from changeling sting. Security doesn't benefit from a cult/rev drink test because they already have a means of deconversion which they will always use over some drink and cultists have their gear

Revs don't start loud, nor does blood cult. Clockies usually go loud but they don't have to. These drinks aren't used as testing and if shitsec would start to do it, I would add bastion brew to all drinks so they would get noted/banned for it. 

The antags are protected by the metashield bit, just because they drink a drink it doesn't mean they can get arrested for it. 
So its fine for the traitors to have a drink that benefits them, it ALSO means that they can make it themselves if they want to. Its honestly not that deep.

[EDIT]

Also, Revs obviously dont start loud, if they did. They would get squashed easily hence why their flash got turned into an implant.

If sec use this as a means to testing for "antags" it will just make it easy for the admins to get rid of the shit-sec. So I see it as a win win

**Review by Moltijoe — APPROVED** [review]

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**